src/main/java/org/greekleanersinc/servicetemplate/client/ClientClient.java

```diff
@@ -0,0 +1,27 @@
+package org.greekleanersinc.servicetemplate.client;
+
+import lombok.extern.slf4j.Slf4j;
+import net.devh.boot.grpc.client.inject.GrpcClient;
+import org.greekleanersinc.servicetemplate.Request;
+import org.greekleanersinc.servicetemplate.Response;
+import org.greekleanersinc.servicetemplate.TemplateServiceGrpc;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ClientClient {
```

Comment:
Medium:
This naming is a bit confusing 🙂  
Can we think about another option?

Reply:
fixed